hardhat/hardhat.config.ts

```diff
@@ -94,9 +96,19 @@ task('mint', 'Mints the given token to specified account')
     }
   );
 
-const forking = {
-  url: ETHEREUM_RPC_URL,
-  blockNumber: FORKED_BLOCK,
+const forking = { url: ETHEREUM_RPC_URL, blockNumber: FORKED_BLOCK };
+
+const hardhatNetwork = {
+  live: false,
+  allowUnlimitedContractSize: true,
+  gas: 'auto' as const,
+  gasPrice: 'auto' as const,
+  gasMultiplier: 1,
+  chainId: 1337,
+  accounts: {
+    mnemonic: 'coordinape',
```

Comment:
We're not using them for anything atm. The most important thing is to set HARDHAT_OWNER_ADDRESS, because the test deployment scripts will fund that address with tokens.